src/app/likes/likes.component.html

```diff
@@ -5,19 +5,33 @@
     [cols]="likes.length > 4 ? 4 : likes.length"
     gutterSize="2"
     rowHeight="1:1"
-    ><a
+  >
+    <mat-grid-tile
       *ngFor="let like of likes"
-      [attr.href]="like.url"
-      target="_blank"
-      title="{{like.title}} by {{like.artist}}"
+      colspan="1"
+      [style.background-image]="'url(' + like.song_art_image_url + ')' | safe: 'style'"
+      style="background-size: cover;"
```

Comment:
prefer CSS to the style attribute (unless it's computed by angular as on the line above)